.github/workflows/pr_validation.yaml

```
@@ -9,12 +9,13 @@ jobs:
         working-directory: ./super_editor
     steps:
       # Checkout the PR branch
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       # Setup Flutter environment
-      - uses: subosito/flutter-action@v1
+      - uses: subosito/flutter-action@v2
         with:
           channel: "stable"
+          architecture: arm64
```

In [here](https://github.com/marketplace/actions/flutter-action) it shows an example where it uses `x64` instead of `arm64`
```suggestion
          architecture: x64
```